diff --git a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
index 00ae69d..304d05f 100644
--- a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
+++ b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
@@ -123,6 +123,7 @@ exports.networks = [
     { chainId: 2020n, shortName: 'pmint' },
     { chainId: 2021n, shortName: 'edg' },
     { chainId: 2039n, shortName: 'aleph' },
+    { chainId: 2187n, shortName: 'g7' },
     { chainId: 2192n, shortName: 'snax' },
     { chainId: 2221n, shortName: 'tkava' },
     { chainId: 2222n, shortName: 'kava' },
@@ -191,6 +192,7 @@ exports.networks = [
     { chainId: 13337n, shortName: 'beam-testnet' },
     { chainId: 13371n, shortName: 'imx' },
     { chainId: 13473n, shortName: 'imx-testnet' },
+    { chainId: 13746n, shortName: 'g7t' },
     { chainId: 17000n, shortName: 'holesky' },
     { chainId: 17069n, shortName: 'garnet' },
     { chainId: 17172n, shortName: 'eclipse' },
